### PR TITLE
✨  add AppsFlyer as amp-analytics vendor

### LINKED ISSUE #35218: add AppsFlyer as amp-analytics vendor
### Summary

We would like to propose an analytics solution for our customers which supports AMP pages. 
We would like to be added to the analytics vendors list.

### Design Document

_No response_

### Motivation

Currently, our customers can't add AppsFlyer's Web-SDK to their AMP pages, so they lose important data about their users. 

### Alternative Solutions

N/A

### Launch Tracker

_No response_

### Notifications

/cc @ampproject/wg-approvers
/cc @ampproject/wg-analytics

### PR DESCRIPTION
**Pull request by fredsh2k**
We would like to propose an analytics solution for our customers which supports AMP pages.
We would like to be added to the analytics vendors list.

Closes #35218 
I2I https://github.com/ampproject/amphtml/issues/35218

Added an OWNERS file as suggested by this issue https://github.com/ampproject/amphtml/issues/35200

### CODE REVIEW

**Comment by fredsh2k** [comment]
hi @alanorozco, please let me know if something else is required for the approval of this PR 😄

**Comment by fredsh2k** [comment]
hi @alanorozco, should I do anything else to proceed with merging this PR?

**Review by alanorozco — APPROVED** [review]

**Comment by alanorozco** [comment]
Would you mind following these steps to unblock [a CI failure](https://app.circleci.com/pipelines/github/ampproject/amphtml/13415/workflows/3250e3e4-1f47-44c4-9d88-8fdb15a89781/jobs/217266)?

```
ERROR: pull/35220 is missing the latest CircleCI config revision e1c2d0fbaf2e37c36e57474f74129601714c3795.


This can be fixed in three ways:


1. Click the "Update branch" button on GitHub and follow instructions.
   ⤷ It can be found towards the bottom of the PR, after the list of checks.


2. Pull the latest commits from main and re-push the PR branch.
   ⤷ Follow these steps:

      git checkout main
      git pull
      git checkout <PR branch>
      git merge main
      git push origin


3. Rebase on main and re-push the PR branch.
   ⤷ Follow these steps:

      git checkout main
      git pull
      git checkout <PR branch>
      git rebase main
      git push origin --force
```

**Pull request closed without merging**
